auth/custom_claims.js

```diff
@@ -27,7 +28,7 @@ admin.auth().verifyIdToken(idToken).then((claims) => {
 // Lookup the user associated with the specified uid.
 admin.auth().getUser(uid).then((userRecord) => {
   // The claims can be accessed on the user record.
-  console.log(userRecord.customClaims.admin);
+  console.log(userRecord.customClaims['admin']);
```

Comment:
Was this switched from `customClaims.admin` to `customClaims['admin']` here and below to make the compiler happy? I think the original `.admin` syntax is nicer

Reply:
Yeah it was ... do you have a suggestion for how we can have the `.` syntax?  I actually don't mind this because it shows a division between types parts `userRecord.customClaims` and the `any` Danger Zone